packages/performance/src/test/performance.test.ts

```diff
@@ -12,4 +16,47 @@ describe('Performance', () => {
     performance.finish();
     expect(spy).toHaveBeenCalledWith(expect.any(Navigation));
   });
+
+  describe.skip('first-input-delay', () => {
```

Comment:
@lemonmade tests aren't runnable right now, but I think they'd pass once a `performance` mock is available.  Can you sanity check, please?